lib/ansible/modules/database/mysql/mysql_user.py

```diff
@@ -649,6 +659,20 @@ def privileges_grant(cursor, user, host, db_table, priv):
     query = ' '.join(query)
     cursor.execute(query, (user, host))
 
+
+def convert_priv_dict_to_str(priv):
+    """Converts privs dictionary to string of certain format.
+
+    Args:
+        priv (dict): Dict of privileges that needs to be converted to string.
+
+    Returns:
+        priv (str): String representation of input argument.
+    """
+    priv_list = ['%s:%s' % (key, val) for key, val in iteritems(priv)]
+
+    return '/'.join(priv_list)
```

Comment:
Does the order matter? If yes, it's probably better to use
```suggestion
    return '/'.join(sorted(priv_list))
```

Reply:
I think the order doesn’t matter